gears/flywheel/lcmodel.json

```diff
@@ -0,0 +1,146 @@
+{
+  "author": "Stephen Provencher",
```

Comment:
I think we usually put the author of the gear here (i.e. Flywheel Exchange, Inc)